src/lsptoolshost/openProjectParams.ts

```diff
@@ -0,0 +1,10 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { DocumentUri } from 'vscode-languageclient/node';
+
+export class OpenProjectParams {
```

Comment:
I think these can go into roslynProtocol.ts, and can just be an interface (doesn't need to be a class)

Reply:
This is the same as openSolutionParams.ts but I agree it can be cleaned up. Do you want me to clean up both?

Reply:
Yes please!

Reply:
Change pushed.